src/OrchardCore.Modules/OrchardCore.OpenId/Views/OpenIdValidationSettings.Edit.cshtml

```diff
@@ -28,6 +28,13 @@
     <span class="hint">@T["The address of the remote OpenID Connect server. Using a HTTPS address is strongly recommended."]</span>
 </div>
 
+<div class="mb-3" asp-validation-class-for="MetadataAddress">
+    <label asp-for="MetadataAddress">@T["Well Known Metadata Address"]</label>
+    <input asp-for="MetadataAddress" class="form-control" />
+    <span asp-validation-for="MetadataAddress"></span>
+    <span class="hint">@T["The address of the metatdata endpoint. Optional. (https://{your base url}/.well-known/openid-configuration)"]</span>
```

Comment:
We need to be clearer about the fact it's only for advanced scenarios where the identity provider doesn't host it at the usual location and that `/.well-known/openid-configuration` will be used by default.

Reply:
Updated